include/aws/mqtt/client.h

```diff
@@ -197,27 +227,45 @@ AWS_MQTT_API
 int aws_mqtt_client_connection_disconnect(struct aws_mqtt_client_connection *connection);
 
 /**
- * Subscribe to a topic filter. on_publish will be called when a PUBLISH matching topic_filter is recieved.
+ * Subscribe to topic filters. on_publish will be called when a PUBLISH matching each topic_filter is recieved.
```

Comment:
received, and shouldn't that be on_suback?

Reply:
I spelled that word wrong A LOT.

No, on_suback is called when a suback is received. on_publish is called when a publish is received.